### ISSUE TITLE
FEEDBACK: give feedback and ideas easily here

### ISSUE THREAD

**Issue by kalaksi** [kalaksi/lightkeeper#14]
This issue is meant to lower the bar for giving feedback and ideas. The goal is to understand how people use, or want to use, Lightkeeper. Are there difficulties and what works well and what doesn't.  
  
You can:
- Report bugs.
- Tell about difficulties in your workflow (the aim of Lightkeeper is to make things easier and more fluent).
- Other kind of ideas or feature requests.
- Give feedback. Maybe there are features you like or dislike?  
  
You can use reactions with existing comments instead of basically repeating the same message to keep the contents of this issue easier to go through.

**Comment by playlogo** [comment]
Hey, so I don't know if this is possible to do with flatpack, but an "import feature" for the `.ssh/config` file would be super nice! Maybe ask the user on first startup, and possibly also import any private keys refrenced in the config file.

**Comment by playlogo** [comment]
Also for some reason some icons look like this for me: 
![image](https://github.com/kalaksi/lightkeeper/assets/62153710/1400670d-9cc7-4e32-9ecd-2d9b55470ca0)
Is my KDE install missing icons ?

**Comment by kalaksi** [comment]
> Hey, so I don't know if this is possible to do with flatpack, but an "import feature" for the .ssh/config file would be super nice! Maybe ask the user on first startup, and possibly also import any private keys refrenced in the config file.

It's possible by using a file selection portal (a dialog). SSH config could be used to help configure lightkeeper, but I wouldn't want to copy any private keys around since that complicates things security-wise. User would have to select the individual private keys using the file portal again.

By the way, ssh-agent works fine with Lightkeeper so that could be a better and easier way to handle keys.

> Is my KDE install missing icons ?

That's interesting... All necessary icons should already be bundled. Are all icons like that? What distro and version are you using? Flatpak or built from source?

**Comment by playlogo** [comment]
> It's possible by using a file selection portal (a dialog). SSH config could be used to help configure lightkeeper, but I wouldn't want to copy any private keys around since that complicates things security-wise. User would have to select the individual private keys using the file portal again.

Yeah, you're right, letting the user select the private keys is also a lot more trustworthy. But I really like the idea of a one-time dialog asking the user to import the config.

> That's interesting... All necessary icons should already be bundled. Are all icons like that? What distro and version are you using? Flatpak or built from source?

I'm running Nobara 39 with KDE Plasma 6.0.5 and the latest Flatpak version. Only some icons are like that, but this could also be an Issue with my linux install/Themes or something else...

**Comment by sdraxler** [comment]
Apologies, if I overlooked something. Isis possible to specify different login users, dependent on the machine? It appears that Lightkeeper tries to use the same account to log into all machines I have configured.

**Comment by kalaksi** [comment]
> Apologies, if I overlooked something. Is it possible to specify different login users, dependent on the machine? It appears that Lightkeeper tries to use the same account to log into all machines I have configured.

Hi, no problem. Currently you would do it by creating a new host group, e.g. "ssh-user2", where you can add the "ssh" connector module to override SSH settings. Then use that host group with hosts you want.
GUI doesn't currently have a more direct way to set those settings per host, but I'm considering of adding something easier so that a separate host group isn't always needed.
Let me know if there are any issues.

**Comment by CocolinoFan** [comment]
Use GPG/PGP key for ssh-ing into servers?
I use key authentication for ssh but is generated from the GPG key. Every time I ssh to a server (through the command line) I get a prompt to enter my PGP password.
I can open a terminal for the service in Lightkeeper terminal emulator and then I am prompted for the GPG password as expected and I connect. But I am only connected/ssh-ed in that terminal emulator window, Lightkeeper still shows `PENDING` and `ssh: Failed to authenticate with SSH agent. `
